Jasmine/Common/Models/GameData.swift

```diff
@@ -1,6 +1,7 @@
 /// Simple container for information necessary at the start of every game
 struct GameData {
 
+    let name: String
```

Comment:
yeah cause I don't want to leak realm outside of level

Question, should I put `points` and `gameStatus` here? Because I realized you guys would have to import realm if you use `LevelResult` object.